clients/web/test/testUtils.js

```diff
@@ -632,17 +632,11 @@ girderTest.promise = $.Deferred().resolve().promise();
  * Import a javascript file and.
  */
 girderTest.addScript = function (url) {
```

Comment:
I'll fix it in another PR. I don't want any more commits at this point, as I'm trying to ensure that the tests repeatably pass.